docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.6'
+volumes:
+  dbdata:
+services:
+  db:
+    image: postgres:11-alpine
+    volumes:
+      - dbdata:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: "password"
+  web:
+    build: .
+    command: /bin/sh -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
+    ports:
+      - 3000:3000
+    depends_on:
+      - db
```

Comment:
As mentioned above, I'd set the 

```yaml
services:
  web:
    environment:
      DATABASE_URL: "postgres://postgres:postgres@db/csc_development"
      RAILS_ENV: "development" # if this is the intention instead of production
```

Reply:
Agree with this.

Reply:
Not sure what the postgres url is on docker, or if it is constant, will revert tomorrow

Reply:
The username is `postgres` and the password is what you set it to in the ENV. The database by default is the same as the username, but could be set by adding `POSTGRES_DB: "csc_development"` and then it would be exactly as I wrote in the example above.